### PR TITLE
Update VigLink

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

File: styles/all/template/event/overall_footer_after.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<!-- IF VIGLINK_ENABLED -->
+{% if VIGLINK_ENABLED %}
 <script type="text/javascript">
 	var vglnk = {
-		key: '{VIGLINK_API_KEY}',
-		sub_id: '{VIGLINK_SUB_ID}'
+		key: '{{ VIGLINK_API_KEY }}',
+		sub_id: '{{ VIGLINK_SUB_ID }}'
```
Comment (kasimi) [inline styles/all/template/event/overall_footer_after.html]: These two values should be escaped.

Comment (VSEphpbb) [reply styles/all/template/event/overall_footer_after.html]: They need to be the key. Escaping them might break it which is why they’ve never been escaped.

Comment (Nicofuma) [reply styles/all/template/event/overall_footer_after.html]: evenwith a javascript escape? (`|e('js')`)

**Comment by VSEphpbb** [comment]
Updated

**Pull request merged**